tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ManualInstrumentation/Tracer/StartActiveSpanCreationSettingsIntegration.cs

```diff
@@ -0,0 +1,45 @@
+﻿// <copyright file="StartActiveSpanCreationSettingsIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.Telemetry.Metrics;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.ManualInstrumentation.Tracer;
+
+/// <summary>
+/// Datadog.Trace.IScope Datadog.Trace.Tracer::StartActive(System.String) calltarget instrumentation
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "Datadog.Trace.Custom",
+    TypeName = "Datadog.Trace.Tracer",
+    MethodName = "StartActive",
+    ReturnTypeName = "Datadog.Trace.IScope",
+    ParameterTypeNames = new[] { ClrNames.String },
```

Comment:
I think you meant to target the SpanCreationSettings overload instead

```suggestion
    ParameterTypeNames = new[] { "Datadog.Trace.SpanCreationSettings" },
```

Reply:
This would also require you to use a generic argument for the parameter in the OnMethodBegin signature

Reply:
Thanks, fixed!